packages/containers/src/ActionBar/ActionBar.container.js

```diff
@@ -49,6 +54,9 @@ const actionPropTypes = PropTypes.oneOfType([
 	}),
 ]);
 
+Object.keys(Component).forEach((key) => {
+	ActionBar[key] = Component[key];
+});
```

Comment:
This is to keep the attached stuff:
Container's ActionBar.SwitchActions , etc ...